bc_obps/service/tests/data_access_service/test_data_access_user_operator_service.py

```diff
@@ -0,0 +1,125 @@
+        approved_operator = baker.make_recipe('utils.operator', status=Operator.Statuses.APPROVED)
+
+        user_operators_with_declined_operator = []
+        approved_admin_user_operators = []
+        pending_admin_user_operators_for_approved_operator = []
+        declined_admin_user_operators = []
+        declined_pending_user_operators = []
+        pending_user_operators_with_pending_status = []
+
+        # Prepare user operators for various roles and statuses
+        # Declined operator with user operators (should be excluded in final result)
+        for _ in range(5):
+            user_operators_with_declined_operator.append(
+                baker.make_recipe(
+                    'utils.user_operator',
+                    role=UserOperator.Roles.ADMIN,
+                    status=UserOperator.Statuses.PENDING,
+                    operator=declined_operator,
+                )
+            )
+
+            # Approved admin user operators (should be included in final result)
+            approved_admin_user_operators.append(
+                baker.make_recipe(
+                    'utils.user_operator', role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.APPROVED
+                )
+            )
+
+            # Pending admin user operators for approved operator (should be included in final result)
+            pending_admin_user_operators_for_approved_operator.append(
+                baker.make_recipe(
+                    'utils.user_operator',
+                    operator=approved_operator,
+                    role=UserOperator.Roles.ADMIN,
+                    status=UserOperator.Statuses.PENDING,
+                )
+            )
+
+            # Declined admin user operators (should be included in final result)
+            declined_admin_user_operators.append(
+                baker.make_recipe(
+                    'utils.user_operator', role=UserOperator.Roles.ADMIN, status=UserOperator.Statuses.DECLINED
+                )
+            )
+
+            # Declined pending user operators (should be included in final result only if no approved admin exists)
+            declined_pending_user_operators.append(
+                baker.make_recipe(
+                    'utils.user_operator', role=UserOperator.Roles.PENDING, status=UserOperator.Statuses.DECLINED
+                )
+            )
+
+            # Pending user operators for the approved operator, with a PENDING status (should be excluded due to approved admin user)
+            pending_user_operators_with_pending_status.append(
+                baker.make_recipe(
+                    'utils.user_operator',
+                    role=UserOperator.Roles.PENDING,
+                    status=UserOperator.Statuses.PENDING,
+                    operator=approved_operator,
+                )
+            )
+
+        # Add approved admin user for the approved operator (to prevent showing pending user operators for this operator)
```

Comment:
I discussed this with Andrea and Zoey, but I forgot to document it in this ticket. Sorry for that. Here’s the conversation: https://github.com/bcgov/cas-registration/issues/2059#issuecomment-2486781705

Reply:
Gotcha, thanks. Internal users don't get to approve/deny after the first admin, but are they supposed to get to view everyone?

Reply:
I believe they prefer to have a minimal number of requests in this table. From what I understand, seeing a record in the table without being able to change it might be annoying for them. Calling @andrea-williams to take radical decisions lol

Reply:
I'm overruling a comment that one SME said (that I believe was just casual and not something they were committing to). IMO it doesn't make sense to show ALL industry admins in that grid when internal users are only expected to action some of them. Internal users can view all users for a specific operator and view the admins from there if they need to.

Boom. Radical Decision made.